docs/source/format/DissociatedIPC.rst

```diff
@@ -0,0 +1,335 @@
+       allow identifying messages by only a portion of the tag, leaving the rest of the
+       bits to be used for control flow or other message metadata.
+
+   Sequence Number
+       A little-endian, 4-byte unsigned integer starting at 0 for a stream, indicating 
+       the sequence order of messages. It is also used to identify specific messages to 
+       tie the IPC metadata header to its corresponding body since the metadata and body
+       can be sent across separate pipes/streams/transports.
+
+       If a sequence number reaches ``UINT32_MAX``, it should be allowed to roll over as
+       it is unlikely there would be enough unprocessed messages waiting to be processed
+       that would cause an overlap of sequence numbers.
+
+       The sequence number serves two purposes: To identify corresponding metadata and 
+       tagged body data messages and to ensure we do not rely on messages having to arrive
+       in order. A client should use the sequence number to correctly order messages as
+       they arrive for processing.
+
+   Backpressure
+       *Currently* this proposal does not specify any way to manage the backpressure of
+       messages to throttle for memory and bandwidth reasons. For now, this will be 
+       **transport-defined** rather than lock into something sub-optimal. 
+
+       As usage among different transports and libraries grows, common patterns will emerge
+       that will allow for a generic, but efficient, way to handle backpressure across
+       different use cases.
+
+       .. note::
+          While the protocol itself is transport agnostic, the current usage and examples 
+          only have been tested using UCX and libfabric transports so far, but that's all.
+
+
+Protocol Description
+====================
+
+A reference example implementation utilizing `libcudf`_ and `ucx`_ can be found at 
+https://github.com/zeroshade/cudf-flight-ucx.
+
+Requirements
+------------
+
+A transport implementing this protocol **MUST** provide two pieces of functionality:
+
+* Message sending
+  - Delimited messages (like gRPC) as opposed to non-delimited streams (like plain TCP 
+    without further framing).
+  - Alternatively, a framing mechanism like the `encapsulated message format <ipc-message-format>`
+    for the IPC protocol can be used while leaving out the body bytes.
+* Tagged message sending
+  - Sending a message that has an attached little-endian, unsigned 64-bit integral tag
+    for control flow. A tag like this allows control flow to operate on a message whose body
+    is on a non-CPU device without requiring the message itself to get copied off of the device.
+
+URI Specification
+-----------------
+
+When providing a URI to a consumer to contact for use with this protocol (such as via 
+the `Location URI for Flight <flight-location-uris>`_), the URI should specify a scheme
+like *ucx://* or *fabric://*, that is easily identifiable. In addition, the URI should
```

Comment:
The `//` is not part of the scheme, it's part of the path (the colon either isn't technically part of the scheme, but I guess it makes the text more easily understood).
```suggestion
like *ucx:* or *fabric:*, that is easily identifiable. In addition, the URI should
```